### ISSUE TITLE
Remove deprecated t.p.text.docstringLStrip

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5036]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5036|
|Type|task|
|Created|2011-04-07 17:03:53Z|

This was deprecated in twisted 10.2 ([#4328](https://github.com/twisted/twisted/issues/4328)) and can be removed.

Attachments:

* [5036.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d81/d81d94c63c94e24b2228b7b422ed4ef43d4413b6/e7336fbc4627ba0296fb66cc726d6c80e89a1501.diff) (2470 bytes) - added by ClearliteX on 2012-03-12 21:04:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5036 5036
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1302195833000000 1302195833000000
changetime__1331594295000000 1331594295000000
version__None None
owner__ClearliteX ClearliteX
cc__thijs cc__ClearliteX
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="ClearliteX's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|ClearliteX commented|
|-|-|

Removed deprecated code and its test.

**Comment by twisted-trac** [comment]
|<img alt="arsenerei's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|arsenerei set owner to ClearliteX|
|-|-|

Looks good to me.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

Thanks for working on this, ClearliteX!  And thank you for reviewing this, arsenerei!

I am merging it now. ([buildbot results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-docstringLStrip-5036))

@ClearliteX - Just two minor nitpicks, for future patches:

 - If imports are no longer used after code is removed, the imports should probably be removed.  In this case in text.py, that would be this line:
```
from twisted.python import deprecate, versions
```

 - A [news file](http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles) should be included for each ticket.

Once again, thank you for working on this!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

Trac commit hooks seems to be broken, so I am just going to close this ticket manually.

Changes have been merged in [33703](http://twistedmatrix.com/trac/changeset/33703) (this link seems to be broken for now - possibly part of the same trac issues).  This should have been the commit message:

```
Merge remove-docstringLStrip-5036: Remove deprecated t.p.text.docstringLStrip

Author: ClearliteX
Reviewer: arsenerei
Fixes: #5036

Apply patch from ClearliteX to remove deprecated twisted.python.text.docstringLS
trip and related tests
```
